tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Order Get Description 5322.feature

```diff
@@ -0,0 +1,43 @@
+﻿Feature: Display the Order Description in an Authority Section
+	As an Authority User
+	I want to view an orders description
+	So that I can ensure that the information is correct
+
+Background:
+	Given Orders exist
+		| OrderId    | Description         | OrderStatusId | Created    | LastUpdated | LastUpdatedBy                        | OrganisationId                       |
+		| C000014-01 | Some Description    | 1             | 11/05/2020 | 11/05/2020  | 335392e4-4bb1-413b-9de5-36a85c9c0422 | 4af62b99-638c-4247-875e-965239cd0c48 |
+		| C000014-02 | Another Description | 2             | 05/05/2020 | 09/05/2020  | a11a46f9-ce6f-448a-95c2-fde6e61c804a | 4af62b99-638c-4247-875e-965239cd0c48 |
+
+@5322
+Scenario: 1. Get an orders description
+	When a GET request is made for an orders description with orderId C000014-01

```

Comment:
Naming stuff is difficult & I tend to struggle with this, but I believe spec flow scenarios should try to represent business requirements. This means less "technical" language e.g orderId. So for example you this be more readable to a BA:

When the user makes a request to retrieve the order description section with the ID <>

Found this [link](https://automationpanda.com/2017/01/30/bdd-101-writing-good-gherkin/) as a good resource.

Reply:
👍